polaris-ui/package.json

```diff
@@ -64,7 +63,7 @@
     "start": "react-scripts start",
     "build-and-serve-local": "./scripts/build-and-serve-local.sh",
     "build": "./scripts/build.sh",
-    "test": "react-scripts test  --watchAll",
+    "test": "react-scripts test --watchAll --transformIgnorePatterns \"node_modules/(?!pdfjs-dist)/\"",
```

Comment:
The associated change to `Layout.tsx` imports `useUserGroupsFeatureFlag.ts`, which has a reference to `useQueryParamsState` which has a reference to `case-search-results.tsx` which has a reference to `case-details.tsx` which eventually imports `pdfjs-dist` via `PdfViewer.ts`.  This upsets jest with the fix being the addition of this flag.

This becomes an issue because of the way the `path` strings are kept with the page tsx files e.g. https://github.com/CPS-Innovation/Polaris/blob/be7c325d8845b9f8f4b53acf4e4f93a9add01b50/polaris-ui/src/app/features/cases/presentation/case-details/index.tsx#L64. If these were kept elsewhere then these long reference trees wouldn't be be pulled in to the unit tests.